src/components/relationships-panel/RelationshipsPanel.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <v-card flat id="relationship-style">
+    <div class="form__row three-column mt-3">
+      <v-card flat rounded="sm" class="relationship_title gray-card px-4">
+        <v-row no-gutters class="align-center mt-5">
+
+          <v-col cols="4">
+            <v-checkbox id="heir-legal-rep-checkbox" class="mt-0" v-model="selectedRelationships"
+              :value="RelationshipTypes.HEIR_LEGAL_REP" :label="RelationshipTypes.HEIR_LEGAL_REP"
+              :rules="relationshipRules" />
+          </v-col>
+
+          <v-col cols="4">
+            <v-checkbox id="officer-checkbox" class="mt-0" v-model="selectedRelationships"
+              :value="RelationshipTypes.OFFICER" :label="RelationshipTypes.OFFICER" :rules="relationshipRules" />
+          </v-col>
+
+          <v-col cols="4">
+            <v-checkbox id="director-checkbox" class="mt-0" v-model="selectedRelationships"
+              :value="RelationshipTypes.DIRECTOR" :label="RelationshipTypes.DIRECTOR" :rules="relationshipRules" />
+          </v-col>
+
+          <v-col cols="4">
+            <v-checkbox id="shareholder-checkbox" class="mt-0" v-model="selectedRelationships"
+              :value="RelationshipTypes.SHAREHOLDER" :label="RelationshipTypes.SHAREHOLDER"
+              :rules="relationshipRules" />
+          </v-col>
+
+          <v-col cols="4">
+            <v-checkbox id="court-ordered-party-checkbox" class="mt-0" v-model="selectedRelationships"
+              :value="RelationshipTypes.COURT_ORDERED_PARTY" :label="RelationshipTypes.COURT_ORDERED_PARTY"
+              :rules="relationshipRules" />
+          </v-col>
+        </v-row>
+      </v-card>
+    </div>
+  </v-card>
+</template>
+<script lang="ts">
+
+import Vue from 'vue'
+import { Component, Watch, Emit, Prop } from 'vue-property-decorator'
+
+@Component({})
+export default class RelationshipsPanelShared extends Vue {
+  /** Draft restoration relationships */
+  @Prop({ default: ['Heir or Legal Representative'] }) readonly draftRelationships!: []
+
+  // Local properties
+  private selectedRelationships = []
+  private relationshipRules = []
+
+  // Relationship Types Enum for the template
+  RelationshipTypes = {
+    COURT_ORDERED_PARTY: 'Court Ordered Party',
+    DIRECTOR: 'Director',
+    HEIR_LEGAL_REP: 'Heir or Legal Representative',
+    OFFICER: 'Officer',
+    SHAREHOLDER: 'Shareholder'
+  }
```

Comment:
You should create a new, shared enum for this (and specify it as a dependency for THIS shared component).

Reply:
Created a relationship-types enum. Added the dependency.